### PR TITLE
fix unclosed rows and error overwriting in cdns.go

### PR DESCRIPTION
**Pull request by DylanVolz**

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/999/
Test PASSed.

File: traffic_ops/traffic_ops_golang/cdn/cdns.go [hunk 1]
```diff
@@ -128,16 +128,17 @@ FROM cdn c`
...
 			return
 		}
+		err := tx.Rollback()
 		if err != nil {
-			tx.Rollback()
-			return
+			log.Errorln(errors.New("rolling back transaction: " + err.Error()))
 		}
-		tx.Commit()
+		return
```
Comment (dangogh) [inline traffic_ops/traffic_ops_golang/cdn/cdns.go]: no need for this return

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1006/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1008/
Test PASSed.

**Pull request merged**